0000-update-glimmer-template-truthiness-semantics.md

```diff
@@ -0,0 +1,106 @@
+---
+stage: accepted
+start-date: 11/16/2022
+release-date: 
+release-versions:
+teams: # delete teams that aren't relevant
+  - framework
+  - learning
+  - steering
+  - typescript
+prs:
+  accepted: # update this to the PR that you propose your RFC in
+project-link:
+---
+
+<!--- 
+Directions for above: 
+
+stage: Leave as is
+start-date: Fill in with today's date, 2032-12-01T00:00:00.000Z
+release-date: Leave as is
+release-versions: Leave as is
+teams: Include only the [team(s)](README.md#relevant-teams) for which this RFC applies
+prs:
+  accepted: Fill this in with the URL for the Proposal RFC PR
+project-link: Leave as is
+-->
+
+# Update Glimmer Template Truthiness Semantics
+
+## Summary
+
+Update truthiness semantics from handlebar's version to javascript's version to allow for typescript to work as users would expect it to
```

Comment:
```suggestion
Provide a path for Glimmer templates to switch away from classic Handlebars truthiness to JavaScript’s truthiness. This will align our templating language with users' expectations and make teaching easier. It will also allow TypeScript features like type narrowing to work as users would expect it to with keywords like `{{and}}` or `{{or}}`.
```